benchmarks/verifications.py

```
@@ -0,0 +1,474 @@
+    table is updated to the same value it was previously set as.
+    '''
+    successful_updates = 0
+    problems = []
+
+    n = 0
+    while n < len(old_worlds) and successful_updates == 0:
+        for i in range(1, 10001):
+            try:
+                entry_id = str(i)
+                if entry_id in old_worlds[n] and entry_id in new_worlds[n]:
+                    if old_worlds[n][entry_id] != new_worlds[n][entry_id]:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)